README.md

```diff
@@ -44,10 +44,10 @@ Sway is an incredible window manager, and certainly one of the most well establi
     - `shadow_inactive_color <hex color with alpha> ex, #0000007F`
 + LayerShell effects (to blur panels / notifications etc):
     - `layer_effects <layer namespace> <effects>`
-    - The current layer namespaces can be shown with `swaymsg -r -t get_outputs | jq '.[0].layer_shell_surfaces | .[] | .namespace'`
+    - The current layer namespaces can be shown with `swayfxmsg -r -t get_outputs | jq '.[0].layer_shell_surfaces | .[] | .namespace'`
     - Example: `layer_effects "waybar" blur enable; shadows enable; corner_radius 6`
       - Note: If an application uses gtk, its namespace is likely to be "gtk-layer-shell"
-    - SwayIPC Example: `swaymsg "layer_effects 'waybar' 'blur enable; shadows enable; corner_radius 6'"`
+    - SwayIPC Example: `swayfxmsg "layer_effects 'waybar' 'blur enable; shadows enable; corner_radius 6'"`
```

Comment:
not sure how I feel about this one. Do you mind if we save the IPC changes for a follow-up PR? I think we need some more discussion there

Reply:
That defeats the whole point of #191.

There's two options:
1. we close this PR and keep SwayFX as a drop-in replacement for Sway
2. we go forward with the rename and make SwayFX independent, allowing it to be installed side-by-side with Sway

If we go forward with the rename, we can't have any packaged files conflicting with upstream. 
Splitting up the changes into more PRs is only going to make it harder to complete the migration.

When it comes to "IPC compatibility", you can temporarily symlink the binaries, e.g.: `~/.local/bin/swaymsg` -> `/usr/bin/swayfxmsg`

As I stated in my earlier comments, I had started working on the packaging side of things. Before the rebase, I had a commit (a06a4ab409592161d398b0f8ca0a86ed86c975ed) that added a subpackage to create those symlinks. This would alleviate the friction when switching between SwayFX and Sway.

From what I understand both @RicArch97 and @WillPower3309 use Nix/NixOS, in which case you can create those symlinks in the `postInstall` step, i.e:

```nix
postInstall = (old.postInstall or "") + ''
  for f in "" msg nag bar; do
    ln -sT "swayfx$f" "$out/bin/sway$f"
  done
'';
```